### PR TITLE
updating instructions for console compilation

### PR DESCRIPTION
**Pull request by nickycutesc**
refer here for more information:

https://github.com/BlueWallet/BlueWallet/issues/802

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,22 @@ cd BlueWallet
 npm install
 ``` 
 
+Please make sure that your console is running the most stable versions of npm and node (even-numbered versions).
```
Comment (bolatovumar) [inline README.md]: "your console" here is not appropriate as you are not running a specific version of console software but rather specific version of Node and npm.

I would adjust this to say something like:

> Please refer to the `engines` field in `package.json` file for the minimum required versions of Node and npm. It is preferred that you use an even-numbered version of Node as these are LTS versions.

I would also place this line above the line that tells the user to run `npm install` as the version of Node and npm matter for the `npm install` command.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,22 @@ cd BlueWallet
...
+If not, please change your node version in the shell, assuming you have installed node and npm via the Homebrew Package Manager.
+
+```
+brew install node@12
+brew unlink node
+brew link node@12
+```
+
+For more information and background information, please refer to this link: https://apple.stackexchange.com/questions/171530/how-do-i-downgrade-node-or-install-a-specific-previous-version-using-homebrew
+
```
Comment (bolatovumar) [inline README.md]: I would remove the above lines (42 to 56) as they are specific to MacOS and require the user to have `brew` installed which not everyone may use. It's best to leave it up to the user how they want to install the required dependencies.

Comment (nickycutesc) [reply README.md]: Noted.

File: README.md (hunk 1 above)
Comment (nickycutesc) [reply README.md]: Noted.

**Comment by nickycutesc** [comment]
I'm not 100% sure why the ci/circleci test is failing, especially since I haven't touched any code @bolatovumar . Should I have made the updates to a new PR?

**Comment by marcosrdz** [comment]
The circle ci failure is related to one of the unit tests we haven’t updated.

**Comment by nickycutesc** [comment]
Ok, so I shouldn't be worried about this error, correct?

**Comment by marcosrdz** [comment]
> Ok, so I shouldn't be worried about this error, correct?

Correct.

**Comment by Overtorment** [comment]
Thanks!

**Pull request merged**